### PR TITLE
fix(security): patch express-jwt to stable candidate

### PR DESCRIPTION
**Pull request by Thenkei**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

**Comment by Thenkei** [comment]
image
Remaining high vulnerabilities.

File: package.json [hunk 1]
```diff
@@ -98,4 +98,4 @@
     "test": "jest",
     "test:coverage": "jest --coverage"
   }
-}
+}
```
Comment (jeffladiray) [inline package.json:101]: Not sure why the newline was removed here

**Comment by Thenkei** [comment]
Looking at the dependencies I see the following ones that shouldn't be there and moved to devDependencies.

`@babel/runtime base32-encode bitwise-xor otplib`

File: package.json (hunk 1 above)
Comment (Thenkei) [reply package.json:101]: My prettier parser... I'll put it back.

**Comment by arnaudbesnier** [comment]
What is the ClickUp task related to this contribution?
Can we have some context about this intent?

**Review by jeffladiray — APPROVED** [review]
LGTM as it is the nearest release to the commit that was referenced via a git link 👍

**Comment by Thenkei** [comment]
> What is the ClickUp task related to this contribution?
> Can we have some context about this intent?
Some client are having issue due to the non-conventional express-jwt we were using (recent regression to fix security issues).

**Pull request merged**